src/configure.js

```diff
@@ -99,9 +103,20 @@ const prompt = async function() {
             type: 'confirm',
             message: 'Would you like WP Local Docker to manage your hosts file?',
             default: currentHosts !== undefined ? currentHosts : defaults.manageHosts,
-        },
+        }
     ];
 
+    if ( fs.exists( path.join( getConfigDirectory(), 'global' ) ) ) {
+        questions.push(
+            {
+                name: 'overwriteGlobal',
+                type: 'confirm',
+                message: 'Do you want to reset your global services configuration? This will reset any customizations you have made.',
+                default: false
+            }
+        );
+    }
+
     const answers = await inquirer.prompt( questions );
 
     return answers;
```

Comment:
When the user runs `10updocker configure` for the first time (or after update this), we don't display the `overwriteGlobal` question, so let's merge the defaults before return so we can create global docker composer in user's home dir.
```suggestion
    return Object.assign( defaults, answers );
```